### PR TITLE
Fix comingEpisodes sorting

### PR DESCRIPTION
**Pull request by OmgImAlexis**
Ref: https://github.com/SiCKRAGETV/SickRage/pull/2766

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

File: gui/slick/views/comingEpisodes.mako [hunk 1]
```diff
@@ -23,6 +23,9 @@
 <%namespace file="/inc_defs.mako" import="renderQualityPill"/>
 <h1 class="header">${header}</h1>
 <div class="h2footer pull-right">
+% if layout != 'poster':
+    ## <button id="popover" type="button" class="btn btn-inline">Select Columns <b class="caret"></b></button> <!-- Not sure why this doesn't work -->
```
Comment (OmgImAlexis) [inline gui/slick/views/comingEpisodes.mako]: For some reason if we try to add this button to comingEpisodes and then the popover js runs we get a warning about a missing config. "Uncaught TypeError: Cannot read property 'config' of undefined".

**Comment by fernandog** [comment]
In progress?

File: gui/slick/views/comingEpisodes.mako [hunk 2]
```diff
@@ -23,6 +23,9 @@
 <%namespace file="/inc_defs.mako" import="renderQualityPill"/>
 <h1 class="header">${header}</h1>
 <div class="h2footer pull-right">
+% if layout != 'poster':
```
Comment (fernandog) [inline gui/slick/views/comingEpisodes.mako]: @OmgImAlexis why not 

% if 'list' == layout:

As line 70

Comment (OmgImAlexis) [inline gui/slick/views/comingEpisodes.mako]: I just copied it from home. We'll look at unifying code later on.

**Pull request merged**